### DISCUSSION TITLE
After flashing, no popups

### DISCUSSION THREAD

**Discussion by Turle123** [eduardofilo/RG350_adam_image#307] [General]
### What happened?

I tried flashing a 16gb card with balenaEtcher but after flashing it and reincerting the card no popups apear and when i try to open it. windows wants me to format the sd card

### Device

RG280V

### Version

v2.1

**Comment by Turle123** [comment]
I figured it out. The SD card needed to be in a fat32 format before i flashed it with the adam_image

**Reply by eduardofilo** [reply]
Pre-formatting should not be necessary. The flashing process builds the partitions in place and formatted as required, as well as the partition table. I don't know what your problem would be. With SD cards and flashing programs there are all kinds of problems. I'm glad it was solved anyway.
